yb-voyager/cmd/analyzeSchema.go

```diff
@@ -433,6 +402,52 @@ func checkStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType string) {
 	}
 }
 
+// Checks Whether there is a GIN index
+/*
+Following type of SQL queries are being taken care of by this function -
+	1. CREATE INDEX index_name ON table_name USING gin(column1, column2 ...)
+	2. CREATE INDEX index_name ON table_name USING gin(column1 [ASC/DESC/HASH])
+*/
+func checkGinVariations(createIndexNode *pg_query.Node_IndexStmt, sqlStmtInfo sqlInfo, fpath string) {
+	indexName := createIndexNode.IndexStmt.GetIdxname()
+	relName := createIndexNode.IndexStmt.GetRelation()
+	schemaName := relName.GetSchemaname()
+	tableName := relName.GetRelname()
+	fullyQualifiedName := lo.Ternary(schemaName != "", schemaName+"."+tableName, tableName)
+	displayObjectName := fmt.Sprintf("%s ON %s", indexName, fullyQualifiedName)
+	if createIndexNode.IndexStmt.GetAccessMethod() != "gin" { // its always in lower
```

Comment:
> always in lower
parser converts it to that?
If there is even a slight possibility, lets use `strings.EqualFold()`

Reply:
as `gin` access method is a keyword it will always be stored lower in the parsed stmt, I have verified that.